save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/tables/TableComponent.kt

```diff
@@ -110,6 +137,31 @@ fun <D : Any> tableComponent(columns: Array<out Column<D, *>>, getData: () -> Ar
                         }
                     }
                 }
+                if (tableInstance.pageCount > 1) {
```

Comment:
Why `> 1`?

Reply:
Because if there is a single page, we don't need pagination controls.